biomage/unstage/unstage.py

```diff
@@ -3,21 +3,151 @@
 import boto3
 import json
 import base64
+import re
 from PyInquirer import prompt
 from github import Github
 
 
-def check_if_exists(org, sandboxid):
-    url = f"https://raw.githubusercontent.com/{org}/iac/master/releases/staging/{sandboxid}.yaml"  # noqa: E501
+def check_if_exists(org, sandbox_id):
+    url = f"https://raw.githubusercontent.com/{org}/iac/master/releases/staging/{sandbox_id}.yaml"  # noqa: E501
 
     s = requests.Session()
     r = s.get(url)
 
     return 200 <= r.status_code < 300
 
 
+def remove_staging_resources(sandbox_id):
+    # Remove staging records
+    dynamodb = boto3.client("dynamodb")
+
+    staged_experiments = dynamodb.scan(
+        TableName="experiments-staging",
+        ProjectionExpression="experimentId",
+        FilterExpression="begins_with(experimentId, :sandbox_id)",
+        ExpressionAttributeValues={":sandbox_id": {"S": sandbox_id}},
+    )
+
+    staged_experiments = [
+        experiment_id["experimentId"]["S"]
+        for experiment_id in staged_experiments.get("Items")
+    ]
+
+    if len(staged_experiments) == 0:
+        return
+
+    staging_tables = [
+        table
+        for table in dynamodb.list_tables().get("TableNames")
+        if re.match(".*-staging", table)
```

Comment:
in several places in the code we refer to names of staging tables and s3 buckets -- might be more convenient to put these in a config file and get their names from that file in the stage and unstage script, so that in the future if we need to rename them, it is less of a pain

Reply:
Hmm.. I can take it out into a `config.yaml` and have the script read values from there.

Reply:
Taken out into config in 9faacf1